### PR TITLE
RMET-3829 :: Add TotalCaloriesBurned variable and read/write ActiveCaloriesBurned permissions

### PR DESCRIPTION
**Pull request by ItsChaceD**
## Description
- Add support for the `TotalCaloriesBurned` health variable.
- Add permissions for `READ_ACTIVE_CALORIES_BURNED` and `WRITE_ACTIVE_CALORIES_BURNED`

## Context
https://outsystemsrd.atlassian.net/browse/RMET-3829

## Type of changes
- [ ] Fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Refactor (cosmetic changes)
- [ ] Breaking change (change that would cause existing functionality to not work as expected)

## Platforms affected
- [x] Android
- [ ] iOS
- [ ] JavaScript

## Checklist
- [x] Pull request title follows the format `RNMT-XXXX <title>`
- [x] Code follows code style of this project
- [x] CHANGELOG.md file is correctly updated
- [x] Changes require an update to the documentation
	- [ ] Documentation has been updated accordingly

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 The changes documented here do not include those from the original repository.
 
+## [2.3.1]
```
Comment (OS-martacarlos) [inline CHANGELOG.md:9]: 2.3.0 has not been released yet , so maybe we can join all of this in the 2.3.0 ?

File: hooks/androidCopyPreferencesPermissions.js [hunk 2]
```diff
@@ -39,6 +39,13 @@ let permissions = {
     },
     CaloriesBurned: {
```
Comment (alexgerardojacinto) [inline hooks/androidCopyPreferencesPermissions.js:40]: Isn't this the same as `ActiveCaloriesBurned`? Do we need to keep both or can we remove `CaloriesBurned` and just keep `TotalCaloriesBurned` and `ActiveCaloriesBurned`?

Comment (ItsChaceD) [reply hooks/androidCopyPreferencesPermissions.js:40]: `CaloriesBurned` (*Both platforms*) = active calories burned only
`TotalCaloriesBurned` (*Android only*) = total calories burned

**Review by alexgerardojacinto — APPROVED** [review]

**Review by OS-pedrogustavobilro — APPROVED** [review]

**Comment by ItsChaceD** [comment]
Closing due to bug with `ActiveCaloriesBurnedRecord` on health connect lib

**Pull request closed without merging**